notebooks/Data_Attribute_Recommendation_Regression_Model_Template.ipynb

```diff
@@ -0,0 +1,1933 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please log in to your trial account: https://cockpit.eu10.hana.ondemand.com/trial/\n",
+    "\n",
+    "In the your global account screen, go to the \"Boosters\" tab:\n",
+    "\n",
+    "![trial_booster.png](attachment:trial_booster.png)\n",
+    "\n",
+    "*Boosters are only available on the Trial landscape. If you are using a production environment, please follow this tutorial to manually [create a service instance and a service key](https://developers.sap.com/tutorials/cp-aibus-dar-service-instance.html)*."
```

Comment:
I am afraid that US10 will not have the boosters right now. I will also check this later.

Reply:
Does this still hold true? I believe we have booster for US10 as well, right?